tests/csm/rest/test_csm_load.py

```diff
@@ -358,23 +359,26 @@ def test_44789(self):
         """
         test_case_name = cortxlogging.get_frame()
         self.log.info("##### Test started -  %s #####", test_case_name)
-        test_cfg = self.test_cfgs["test_44789"]
-        jmx_file = "CSM_Create_N_Monitor_Create_N_IAM_Set_Quota.jmx"
-        self.log.info("Running jmx script: %s", jmx_file)
 
         resp = self.csm_obj.list_iam_users_rgw()
         assert resp.status_code == HTTPStatus.OK, "List IAM user failed."
         user_data = resp.json()
         self.log.info("Step 1: List user response : %s", user_data)
+        existing_user = len(user_data['users'])
+        self.log.info("Existing iam users count: %s", existing_user)
+        self.log.info("Max iam users : %s", rest_const.MAX_IAM_USERS)
+        new_iam_users = rest_const.MAX_IAM_USERS - existing_user
+        self.log.info("New users to create: %s", new_iam_users)
 
-        result = self.jmx_obj.run_verify_jmx(
-            jmx_file,
-            threads=test_cfg["threads"],
-            rampup=test_cfg["rampup"],
-            loop=test_cfg["loop"])
-        assert result, "Errors reported in the Jmeter execution"
+        self.log.info("Step 2: Create IAM users in parallel with Set Quota")
+        result = self.csm_obj.create_multi_iam_user_set_quota(new_iam_users, existing_user)
+        assert result, "Unable to create users"
```

Comment:
without this assert, test is passing

Reply:
[2022-09-08 01:46:56] [MainThread] [INFO  ] [test_csm_load.py: 124]: CSM user deleted []
[2022-09-08 01:46:56] [MainThread] [INFO  ] [test_csm_load.py: 138]: Done: Teardown completed.
======================================================= short test summary info =======================================================
PASSED tests/csm/rest/test_csm_load.py::TestCsmLoad::test_44789
============================================== 1 passed, 4 warnings in 870.68s (0:14:30) ==============================================
(virenv) [root@ssc-vm-g2-rhev4-1909 cortx-test]#